internal/cmd/server_test.go

```diff
@@ -141,6 +116,152 @@ func TestServerCmd_LoadOptions(t *testing.T) {
 				return expected
 			},
 		},
+		{
+			name: "all options from config",
+			setup: func(t *testing.T, cmd *cobra.Command) {
+				content := `
+version: 0.2
+tlsCache: /cache/dir
+enableTelemetry: false # default is true
+enableSignup: false    # default is true
+sessionDuration: 3m
+
+dbFile: /db/file
+dbEncryptionKey: /this-is-the-path
+dbEncryptionKeyProvider: the-provider
+dbHost: the-host
+dbPort: 5432
+dbName: infradbname
+dbUsername: infra
+dbPassword: env:POSTGRES_DB_PASSWORD
+dbParameters: sslmode=require
+
+keys:
+  - kind: vault
+    config:
+      token: the-token
+      address: 10.1.1.1:1234
+
+secrets:
+  - kind: env
+    name: base64env
+    config:
+      base64: true
+
+addr:
+  http: "1.2.3.4:23"
+  https: "1.2.3.5:433"
+  metrics: "1.2.3.6:8888"
+
+ui:
+  enabled: false # default is true
+  proxyURL: "1.2.3.4:5151"
+
+providers:
+  - name: okta
+    url: https://dev-okta.com/
+    clientID: client-id
+    clientSecret: the-secret
+
+grants:
+  - user: user1
+    resource: infra
+    role: admin
+  - group: group1
+    resource: production
+    role: special
+
+users:
+  - name: username
+    accessKey: access-key
+    password: the-password
+
+# TODO: config providers/grants/users
```

Comment:
```suggestion
```

Looks like this is actually in the test config

Reply:
Good catch! I forgot to remove the command after adding them